source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -94,12 +94,18 @@ To set up the metrics exporter app:
 
 4. Run `cf start metric-exporter` to start your app
 1. Check you're generating Prometheus metrics at the metrics endpoint
-    - `https://<your-app-url>/metrics`
+    - `https://metric-exporter.cloudapps.digital/metrics`
```

Comment:
only one team can have this URL.  Maybe something like `https://<app-name>.cloudapps.digital/metrics`?